src/main/java/com/blogify/blogapi/model/User.java

```diff
@@ -39,30 +39,7 @@ public class User implements Serializable {
   @Getter(AccessLevel.NONE)
   private Instant creationDatetime;
```

Comment:
Add `@CreationTimestamp`

Reply:
@CreationTimestamp is already there, but above the @Getter

Reply:
it's for delete methode getter in the creationDatetimen. i want to dele setter heare but misklic. because creationDatetime is create automaticly in ccreation un can't be modify after

Reply:
it's good with setter(... , but mybe, i'll delete it to be easier